src/Sarif.FunctionalTests/SarifConverterTests.cs

```diff
@@ -91,7 +91,7 @@ public void TSLint_EndToEnd()
             BatchRunConverter(ToolFormat.TSLint, "*.json", TestMode.CompareFileContents);
         }
 
-        [Fact(Skip = "BadImageFormatException introduced in net core port needs to be fixed")]
+        [Fact]
         public void PREfastConverter_EndToEnd()
         {
             BatchRunConverter(ToolFormat.PREfast, TestMode.CompareObjectModels);
```

Comment:
Does this still need to be compared this way? If it can be compared by CompareFileContents, maybe we should just do it that way all the time and remove the while TestMode stuff, to keep things clean?

Reply:
Done

---
In reply to: [146696755](https://github.com/Microsoft/sarif-sdk/pull/657#discussion_r146696755) [](ancestors = 146696755)